src/components/StudentFeedback.vue

```diff
@@ -23,9 +23,38 @@
         </p>
       </div>
     </div>
+    <div v-else>
+      <h1 class="s-h1">There is nothing for this module yet. Stay tuned.</h1>
+    </div>
 
     <section class="notes-container">
-      <h2>Notes section goes here</h2>
+      <p class="feedback-container__category--label">
+        <span class="s-h2">Student Notes</span>
+      </p>
+      <div class="current-notes">
+        <ul>
+          <li v-for="(note, index) in studentNotes" :key="index">{{ note }}</li>
+        </ul>
+      </div>
+
+      <div class="form-container">
+        <button @click="toggleNoteForm" class="s-button-secondary show form-container__item">{{ showNoteForm ? 'Hide form' : 'Add new note' }}</button>
+        <form @submit.prevent="AddNote" v-if="showNoteForm" class="form-container__item">
```

Comment:
Is this similar to `event.preventDefault()` ?

Reply:
Yep!  same thing!